src/partials/_head.hbs

```diff
@@ -9,4 +9,5 @@
 
 	grunticon(["/icons/icons.data.svg.css", "/icons/icons.data.png.css", "/icons/icons.fallback.css"], grunticon.svgLoadedCallback);
 </script>
+<script src="/dist/index.js"></script>
```

Comment:
This script tag should be loaded at the bottom of the page, right before the closing body tag.